vllm/transformers_utils/tokenizer_group/ray_tokenizer_group.py

```diff
@@ -0,0 +1,147 @@
+import asyncio
+import os
+from typing import List, Optional
+
+from transformers import PreTrainedTokenizer
+
+from vllm.config import TokenizerPoolConfig
+from vllm.lora.request import LoRARequest
+from vllm.engine.ray_utils import ray
+from vllm.transformers_utils.tokenizer_group.base_tokenizer_group import (
+    BaseTokenizerGroup)
+from vllm.transformers_utils.tokenizer_group.tokenizer_group import (
+    TokenizerGroup)
+from ray.util.scheduling_strategies import NodeAffinitySchedulingStrategy
+
+RayTokenizerGroup = ray.remote(TokenizerGroup)
+
+
+class RayTokenizerGroupPool(BaseTokenizerGroup):
```

Comment:
Why not subclass `TokenizerGroup` here? Then no need for `self.tokenizer` or the `get_lora_tokenizer`, `get_lora_tokenizer_async` overrides. (I'd made this change [here](https://github.com/njhill/vllm/compare/57b419697faee64368c93200806171ae3db9a38a...async_tokenization#diff-cd93439ab906c39ae8ad7e31cbc49ec37e5590cf4ec65a84b1de048fcaf54349L222))

Reply:
yeah we can do that